lib/europeana/feed_jobs.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'active_job'
+require 'feedjira'
+require 'europeana/feed_jobs/engine'
+
+module Europeana
+  module FeedJobs
```

Comment:
I'm not convinced this is a great name for the Engine/gem/repo because there is already more than just the job in here. I think `Europeana::Feeds` would be better.